client/landing/stepper/declarative-flow/internals/steps-repository/plans/plans-wrapper.tsx

```diff
@@ -105,6 +115,10 @@ const PlansWrapper: React.FC< Props > = ( props ) => {
 			return renderLoading();
 		}
 
+		if ( isNewsletterFlow( flowName ) ) {
+			planTypes = [ TYPE_FREE, TYPE_PERSONAL, TYPE_PREMIUM ];
```

Comment:
Just to confirm: I know I suggested this [in my lastest comment](https://github.com/Automattic/wp-calypso/pull/75174/#issuecomment-1524849986). However, are we sure we don't need to put `TYPE_BUSINESS` in? I'm asking because I just realized that in the screenshot of the PR description there is actually the Business plan.

Reply:
Yes, Lucas said that `Pablo pushed back the pricing page with 4 plans` [here](https://github.com/Automattic/martech/issues/1533#issuecomment-1494934081) :) Also, updated the PR image!